apps/tests/tests/s3-sse-enabled.test.ts

```diff
@@ -0,0 +1,32 @@
+import { describe, expect, it } from 'vitest';
+import ServiceSideEncryptionEnabled from '../../../packages/sls-mentor/src/rules/serverSideEncryptionEnabled';
+import { FAIL_INTELLIGENT_TIERING_BUCKET_NAME } from '../lib/failStack/s3';
+import { PASS_INTELLIGENT_TIERING_BUCKET_NAME } from '../lib/passStack/s3';
+import { slsMentorResult } from './testSetup/slsMentorResult';
+
+const ruleName = ServiceSideEncryptionEnabled['ruleName'];
+
+const slsMentorOutput = slsMentorResult[ruleName];
+
+describe('s3-sse-enabled', () => {
+  it('sls-mentor passes on S3 bucket with SSE enable ', () => {
+    const { result } = slsMentorOutput;
+    expect(
+      result.find(r =>
+        r.arn.resource.includes(
+          PASS_INTELLIGENT_TIERING_BUCKET_NAME.toLowerCase(),
```

Comment:
to be discussed but maybe it would be good if each rule had its dedicated pass and fail resource(s).

Either way, if not, the variable name for logical ID needs to be changed